pkg/ccl/backupccl/backup_cloud_test.go

```diff
@@ -172,6 +172,40 @@ func TestCloudBackupRestoreAzure(t *testing.T) {
 	backupAndRestore(ctx, t, tc, []string{uri.String()}, []string{uri.String()}, numAccounts, nil)
 }
 
+// TestCloudBackupRestoreAzureImplicitAuth hits the real Azure Blob Storage
+// and so could occasionally be flaky. It's only run if the AZURE_ACCOUNT_NAME
+// and AZURE_CONTAINER environment vars are set.
+func TestCloudBackupRestoreAzureImplicitAuth(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+	accountName := os.Getenv("AZURE_ACCOUNT_NAME")
+
+	// NB: the Azure Account key must not be url encoded.
+	accountKey := os.Getenv("AZURE_ACCOUNT_KEY")
```

Comment:
aha, so if implicit auth only accepts rbac, we should be checking for different env vars no?

Reply:
sure, i'll change it to only check name and container. Those are needed on all the tests because, recall, they uniquely define a single bucket 😃